logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/MessageSender.kt

```diff
@@ -479,4 +492,8 @@ internal class MessageSenderImpl internal constructor(
             messageRepository.persistRecipientsDeliveryFailure(message.conversationId, message.id, messageSent.failedToConfirmClients)
         }
 
+    companion object {
+        const val MAX_RETRY_COUNT: Int = 1
```

Comment:
Question: The same as prev. comment about uniformity for same concept, why 1 and not 2 ? I believe there is not right or wrong, since they are just arbitrary constants, but leaving here the question anyway :D

Reply:
1 retry should be enough because we send the message and handle all the errors and retry, if that failed then any other retry will also fail